### ISSUE TITLE
Create Person

### ISSUE THREAD

**Issue by CarsonF** [SeedCompany/cord-field#223]
Follow suit of `CreateOrganization` to `CreatePerson` (should actually be called _Person_ here since we _create people_ but _register users_)
- Should have fields for every property on `CreatePerson` gql type
- Required/Optional should follow gql requirements

Blocked by https://github.com/SeedCompany/cord-api-v3/issues/444

**Comment by glassblowerscat** [comment]
@CarsonF What do we want to do with the phone field in this form? We don't appear to have a `PhoneInput`, and I had started to go down the path of using `material-ui-phone-number`, but there are a couple issues with it:

- The masking breaks if I just use our kind of default implementation of `useField` with it. This decreases the utility of using a custom component, since the user will have more latitude for mistakes
- The `<select>` that loads all the country options is pretty slow; it's not a great experience
- This library doesn't give us validation out of the box

I wasn't sure what our requirements were for this, so I figured it'd be better to check in with you before putting in a lot of work on it.

**Comment by CarsonF** [comment]
To get it in we can just use a `TextField`. Phone numbers are hard to get right, so we want to default to the side of user freedom here. However, having a smarter input is desired eventually.

I've never found those wrapper libraries useful as they solve one aspect but not others. I always end up writing my own.
I've got a `FormattedTextField` in progress that could be used as the foundation for this. Would just need to be connected to libphonenumber. 
If we also wanted to do a country select then that would need an autocomplete field. And would make things a bit more complicated since FF state would probably not be enough. 

All in all, it's a hard problem with little user gain, so it's low on my priority list. Since it's very similar to formatted text and autocomplete functionality it should probably wait until after I finish those two fields early next week. errr this week.

**Comment by glassblowerscat** [comment]
Okay, that's kind of where I was as well. I've already done the `TextField` version, so I think this is just waiting on API blockers for me to finish it off
